### ISSUE TITLE
Where is Japanese .lang & .short files?

### ISSUE THREAD

**Issue by nassau-tk** [cizia64/CrossMix-OS#127]
I want fix the translation those files but I can't find them.

Please tell me about those files place.

**Comment by nassau-tk** [comment]
[jp.zip](https://github.com/user-attachments/files/16342038/jp.zip)

I made from en.lang file.
*test file

But, .lang files will not loading the OS?

**Comment by cizia64** [comment]
you have replaced the files in `/usr/trimui/res/lang` ?

**Comment by nassau-tk** [comment]
Yes, I replaced en.lang and my jp.lang(named on en.lang)
But, It was not changed the words.
Off course, I set the language English.

Is there way to test for changed language?

**Comment by nassau-tk** [comment]
I want to do more check & test to fix the translation on my device.

**Comment by cizia64** [comment]
I don't get it, you have to replace the files :
/usr/trimui/res/lang/jp.lang
and
 /usr/trimui/res/lang/jp.lang.short

Then reboot or restart MainUI and it should be updated (it's not on the SD card but on the internal storage, use SMB or FTP to access it)

**Comment by nassau-tk** [comment]
Oh! 
Internal storage!!
I was seraching in SD.lol
OK!! I got fixing Japanese translation.
And make CJK Font.
If Replace full.ttf for my made it then Japanese characters will appear correctly.
*2Type-Chinese & Hungle & Japanese full set font.
Notice: File size 27MB.

I attached those files for your reference.

[TSP_jp_translationSet.zip](https://github.com/user-attachments/files/16464299/TSP_jp_translationSet.zip)
